editions/test/tiddlers/tests/test-checkbox-widget.js

```diff
@@ -234,6 +234,38 @@ Tests the checkbox widget thoroughly.
             },
         ];
 
+        // https://github.com/Jermolene/TiddlyWiki5/issues/6871
+        const listModeTestsWithListField = (
```

Comment:
@Jermolene ... Should we allow const, arrow functions and ... rest in tests. .. **I'd say yes**, because our test equipment should be new enough ;)

Reply:
I've actually already used `const` and arrow functions in the rest of the test-checkbox-widget.js code from #6561. So if that worked, then this will also work. If `const` and arrow functions in the tests are going to cause problems, the problems would probably have shown up already. :-)

Reply:
I think we should avoid new-fangled JS in the test suite because it runs in the browser, and we want to be able to run our tests in the oldest browsers that we support.

Reply:
So do you want me to remove the `const` and arrow-function code that has already been released in 5.1.23? Because I wasn't aware of that requirement when I wrote https://github.com/Jermolene/TiddlyWiki5/pull/6561, so I included some uses of `const` and arrow-function definitions. I thought that the unit tests only run in Node environments, and that therefore I didn't need to be careful about IE support in the `editions/test` directory tree. (I was careful not including any newfangled JS in the main code, of course).

Reply:
I think it would be worth removing them in a separate PR, if you can. Much appreciated thank you